src/Sanitizer.php

```diff
@@ -63,7 +66,6 @@ class Sanitizer
 
     /**
      * @var DOMDocumentFragment
-     * @deprecated since v2.1.0, not required anymore
```

Comment:
Keep the deprecations with `deprecated since v1.5.0`

Reply:
Done